sesam.py

```diff
@@ -714,7 +725,7 @@ def upload(self):
 
         self.logger.info("Config uploaded successfully")
 
-        if self.args.post_testdata and os.path.isdir("testdata"):
```

Comment:
Should also remove parsing of this argument in the main function if we're going to remove it

Reply:
fixed :)